ddtrace/contrib/botocore/__init__.py

```diff
@@ -2,19 +2,34 @@
 The Botocore integration will trace all AWS calls made with the botocore
 library. Libraries like Boto3 that use Botocore will also be patched.
 
-This integration is automatically patched when using ``patch_all()``::
+Enabling
+~~~~~~~~
 
-    import botocore.session
-    from ddtrace import patch
+The botocore integration is enabled automatically when using
+:ref:`ddtrace-run<ddtracerun>` or :ref:`patch_all()<patch_all>`.
+
+Or use :ref:`patch()<patch>` to manually enable the integration::
 
-    # If not patched yet, you can patch botocore specifically
+    from ddtrace import patch
     patch(botocore=True)
 
-    # This will report spans with the default instrumentation
-    botocore.session.get_session()
-    lambda_client = session.create_client('lambda', region_name='us-east-1')
-    # Example of instrumented query
-    lambda_client.list_functions()
+Configuration
+~~~~~~~~~~~~~
+
+.. py:data:: ddtrace.config.botocore['distributed_tracing']
+
+   Whether to inject distributed tracing data to requests in SQS and Lambda.
```

Comment:
Let's also mention the environment variable that can be used (`DD_BOTOCORE_DISTRIBUTED_TRACING`)

Reply:
I added `DD_TRACE_BOTOCORE_DISTRIBUTED_TRACING`  instead of `DD_BOTOCORE_DISTRIBUTED_TRACING`.

Reply:
Actually, after reading the code, I think you were right. I would need to change the one here too. https://github.com/DataDog/dd-trace-py/blob/master/ddtrace/contrib/starlette/__init__.py#L35

Reply:
ah good catch!